README.md

```diff
@@ -355,8 +384,16 @@ function App() {
 
 ## Contributing
 
-Before submitting a pull request, please read our [Contributing Guide](.github/CONTRIBUTING.md). It contains a set of instructions and guidelines that aim to make the contribution process easy and frictionless.
+Thanks for being interested on making this package better. We encourage everyone to help improving this project with some new features, bug fixes and performance issues. Please take a little bit of your time to read our guides, so this process can be faster and easier.
+
+### Contributing Guide
+
+Take a moment to read about our [contributing guidelines](/.github/CONTRIBUTING.md) so you can understand how to submit an issue, commiting and pull requests.
+
+### Code of Conduct
+
+We expect you to follow or [Code of Conduct](/.github/CODE_OF_CONDUCT.md). Here you can read it and understand what kind of behaviour will not be tolerated.
 
-## License
+### License
```

Comment:
I really don't think that `License` should be a sub-item of `Contributing`. While it does play a role, the license is not limited to the scope of contributions.